src/components/InstallerView.jsx

```diff
@@ -0,0 +1,31 @@
+import viewStyles from '../styles/view'
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/helpers/i18n'
+import classNames from 'classnames'
+
+class InstallerView extends Component {
+  doInstall () {
+    // get the values from the fields
+    const slug = document.querySelector('#slug').value
+    const repourl = document.querySelector('#repourl').value
+    const isupdate = document.querySelector('#isupdate').checked
+    this.onSubmit(slug, repourl, isupdate)
+  }
+
+  render ({t, onSubmit}) {
+    this.onSubmit = onSubmit
+    return (
+      <div role='contentinfo'>
+        <div className={classNames(viewStyles['set-view-content'], viewStyles['set-view-content--narrow'])}>
+          <h2 className={viewStyles['set-view-title']}>{t('InstallView.title')}</h2>
+          <input id='slug' placeholder='slug' /><br />
```

Comment:
As we already speak we should use the `Input` component here, it should not be very expensive.